README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
+
+- Companies involve many people, so it’s important to track how much of the company each person owns, so that future profits are divided appropriately. Historically, this has been done a few ways, depending on the size and nature of the company. One way to do this, especially when a lot of people are involved, is via stock, which we’ll get into next.
+
+- Companies in the US include sole proprietorships, partnerships, limited liability companies (LLCs), S corporations, and C corporations.
+ 	- 🔸 This Guide does not cover equity compensation in LLCs, as they are rarely used as the choice of entity for technology startups, and equity compensation in LLCs is dramatically different from equity compensation in corporations. So we’ll focus on the one type of company most readers of this Guide will be concerned with: The C corporation.
+
+- A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is a type of business entity in the United States. Specifically, it’s a kind of [stock corporation](https://en.wikipedia.org/wiki/Joint-stock_company) with certain federal tax treatment. Most large companies you’ve heard of, as well as startups that aspire to be large, are C corporations. In practice, for a [few](https://www.nytimes.com/2012/07/01/business/how-delaware-thrives-as-a-corporate-tax-haven.html) [reasons](https://www.quora.com/Why-do-most-technology-startups-incorporate-in-Delaware), these companies are usually formed in Delaware, so legalities of all this are defined in Delaware law.
+
+### What is stock and what is equity?
+
+- [**Stock**](https://en.wikipedia.org/wiki/Stock) is a legal invention that helps track ownership of a company. It allows the company to grant ownership to a variety of people or other companies in flexible ways. Stock is divided into **shares**, and every owner, or **shareholder**, holds a specific number of shares. Typically, founders, investors, employees, board members, contractors, advisors, and other companies, like law firms, can all be shareholders.
+
+- Stock ownership is formalized on **stock certificates**, which are fancy pieces of paper that show you own the stock. Sometimes you have stock but don’t have the physical certificate, as it may be held for you at a law office.
+
+- [**Outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) refer to the total number of shares at a given point, reflecting how many shares are currently held by all shareholders. This number starts at an essentially arbitrary value (such as 10 million) when the company is created, and thereafter will increase as new shares are added (issued) and granted to people in exchange for money or services. Outstanding shares may increase or decrease for other reasons, too (such as stock splits and share buyback, which we won’t get into here). There is a lot of nuance and complexity in all this, because each part is defined by piles and piles of legal documents.
```

Comment:
Edit: should be "stock splits and share buybacks"

Reply:
✔️